crates/ruff_db/src/vendored.rs

```diff
@@ -112,6 +148,20 @@ impl fmt::Debug for VendoredFileSystem {
     }
 }
 
+impl Default for VendoredFileSystem {
+    fn default() -> Self {
+        let mut bytes: Vec<u8> = Vec::new();
+        let mut cursor = Cursor::new(&mut bytes);
+
+        {
+            let mut writer = ZipWriter::new(&mut cursor);
+            writer.finish().unwrap();
+        }
+
+        VendoredFileSystem::new(bytes).unwrap()
+    }
+}
```

Comment:
In https://github.com/astral-sh/ruff/pull/11863#discussion_r1642716996, you said you "strongly" preferred not to implement `Default` for this struct ;) What made you change your mind on this? :)

Reply:
Haha. I guess the only reason is that I needed an empty `VendoredFileSystem` to construct `TestDb`s

Reply:
I guess I'm just not _necessarily_ sure it needs to be a field on `(Test)Db` at all

Reply:
There are tests in `ruff_db` where it has to be a field.